CDV-diner-project/Dishes.json

```diff
@@ -0,0 +1,233 @@
+{
+  "soups": [
```

Comment:
Następnie jeżeli weźmiemy to jako tablice to w środku tej tablicy zrobiłbym obiekty czyli 
`[
  {
    categoryName: "Zupy",
    categoryValues: [
     Tutaje te dania
    ]
  }, ...
]`